server/src/routers/auth.ts

```diff
@@ -9,4 +9,10 @@ authRouter.post("/register", register);
 // Endpoint to login a user.
 authRouter.post("/login", login);
 
+//forgot-password
+authRouter.post("/forgot-password", forgotPassword);
```

Comment:
Use authentication middleware for authenticated routes as these route can't be managed by unauthenticated users. Refer passwords route.

It should like:
`passwordRouter.get("/passwords/", isAuthenticated, getAllPasswords);`